client/components/forms/form-password-input/style.scss

```diff
@@ -1,33 +1,24 @@
 .form-password-input {
 	position: relative;
 
-	.form-text-input {
+	.form-text-input[type] {
 		/*!rtl:ignore*/
-		padding-right: 32px;
+		padding-right: 36px;
```

Comment:
Allows more room for password manager UI

<img src="https://github.com/user-attachments/assets/51cf90c7-fbc0-4fba-9bc6-026622d4c72b" width="500">